fix(docs): replace links to concerto from Hyperledger to Accord repo

daniloff200 opened the pull request:
# Issue #90

### Changes
Replace Concerto links

closes #90

mttrbrts commented on website/pages/en/index.js:
```diff
@@ -164,7 +164,7 @@ const Model = props => (
     {[
       {
-        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
+        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/accordproject/concerto">Composer Concerto Modeling Language</a>.</div>',
```
Can you remove the reference to Concerto on this line, please?

daniloff200 replied:
hello !
do you mean complete removal of `<a></a>`, or, just some hyperlink fix?

daniloff200 replied:
@mttrbrts  ping :)

irmerk replied:
I believe @mttrbrts is saying to remove the second sentence "_Accord Project templates use the ..._"

daniloff200 replied:
@irmerk  well, sounds reasonable

gonna do that then

mttrbrts commented on website/versioned_docs/version-0.10.0/accordproject-specification.md:
```diff
@@ -97,7 +97,7 @@ The first step in converting the late delivery and penalty clause to use Accord
 These data elements comprise the Template Model for the clause. The template model is critical in that it defines formal semantics for the clause, and it is a locale neutral representation of the data that a template requires. It also enables powerful search, filtering and organization of templates, for example by finding all templates related to concept X, or all templates that can process a req ...
 
-They are captured formally using the [Concerto modeling language](https://github.com/hyperledger/composer-concerto) (CML). CML is a lightweight schema language that defines namespaces, types, and relationships between types. It includes first-class support for modeling participants (individuals or companies), assets, transactions, enumerations, concepts, and events, and includes the typical featu ...
+They are captured formally using the [Concerto modeling language](https://github.com/accordproject/concerto) (CML). CML is a lightweight schema language that defines namespaces, types, and relationships between types. It includes first-class support for modeling participants (individuals or companies), assets, transactions, enumerations, concepts, and events, and includes the typical features of  ...
```
This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.

mttrbrts commented on website/versioned_docs/version-0.10.0/ergo-intro-examples.md:
```diff
@@ -23,7 +23,7 @@ This declares that this module belongs to the `org.accordproject.helloworld` nam
 It also declares that the contract `HelloWorld` is parameterized over a given `TemplateModel`.
 
-The `TemplateModel` as well as the `Request` and `Response` are types which are specified using [Composer Concerto modeling language](https://github.com/hyperledger/composer-concerto).
+The `TemplateModel` as well as the `Request` and `Response` are types which are specified using [Composer Concerto modeling language](https://github.com/accordproject/concerto).
```
This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.

mttrbrts commented on website/versioned_docs/version-0.10.0/ergo.md:
```diff
@@ -29,7 +29,7 @@ To achieve those goals the design of Ergo is based on the following
 
 - Ergo contracts have a class-like structure with clauses akin to methods
-- Ergo can handle types (concepts, transations, etc) defined with the [Composer Concerto Modeling Language](https://github.com/hyperledger/composer-concerto) (so called CML models), as mandated by the Accord Project Template Specification
+- Ergo can handle types (concepts, transations, etc) defined with the [Composer Concerto Modeling Language](https://github.com/accordproject/concerto) (so called CML models), as mandated by the Accord Project Template Specification
```
This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.

mttrbrts commented on website/versioned_docs/version-0.10.1/accordproject-specification.md:
```diff
@@ -97,7 +97,7 @@ The first step in converting the late delivery and penalty clause to use Accord
 These data elements comprise the Template Model for the clause. The template model is critical in that it defines formal semantics for the clause, and it is a locale neutral representation of the data that a template requires. It also enables powerful search, filtering and organization of templates, for example by finding all templates related to concept X, or all templates that can process a req ...
 
-They are captured formally using the [Concerto modeling language](https://github.com/hyperledger/composer-concerto) (CML). CML is a lightweight schema language that defines namespaces, types, and relationships between types. It includes first-class support for modeling participants (individuals or companies), assets, transactions, enumerations, concepts, and events, and includes the typical featu ...
+They are captured formally using the [Concerto modeling language](https://github.com/accordproject/concerto) (CML). CML is a lightweight schema language that defines namespaces, types, and relationships between types. It includes first-class support for modeling participants (individuals or companies), assets, transactions, enumerations, concepts, and events, and includes the typical features of  ...
```
This is a static versioned file and shouldn't be edited, please remove this change.

When we make a new release of the documentation, Docusarus creates these `versions_docs` automatically as a snapshot of the documentation for historical reference.

irmerk commented on website/pages/en/index.js:
```diff
@@ -164,7 +164,7 @@ const Model = props => (
     {[
       {
-        content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
+          content: '<div class="typeset">The template model defines the structure of data that are used in the natural language and logic. Accord Project templates use the <a href="https://github.com/hyperledger/composer-concerto">Composer Concerto Modeling Language</a>.</div>',
```
I think this link still needs to be changed.

daniloff200 replied:
well
If I remove this sentence, than, I think not :D

irmerk replied:
AH! Yes, I think it's still early for me, I was a bit confused.

irmerk replied:
@mttrbrts I think just still waiting on confirmation on the removal of the second sentence.

jeromesimeon replied:
I'm confused by this thread, I don't see a sentence removed and it still uses the old link and name for Concerto.

daniloff200 replied:
@jeromesimeon the main question was : should it be removed completely, or not? I revert all changes back before getting the final requirements, about the name - yes, that will be done

jeromesimeon commented on docs/example-eatapple.md:
```diff
@@ -22,7 +22,7 @@ variables such as `employee`, `company` and `tax`.
 
 The second component of a smart legal template is the model, which is
 expressed using the [Composer Concerto modeling
```
The name for it should also be changed. Please remove`Composer` everywhere referring to the Concerto modelling language.

daniloff200 commented:
Well, okay :)
So small tasks, but, so many comments :) 

I removed sentence, mentioned by @mttrbrts & @irmerk 
Also, removed  Composer everywhere referring to the Concerto modelling language.

irmerk commented on docs/basic-templates.md:
````diff
@@ -91,7 +91,7 @@ transaction MyResponse extends Response {
 ```
 
-The `TemplateModel` as well as the `Request` and `Response` are types which are specified using the [Composer Concerto modeling language](https://github.com/hyperledger/composer-concerto).
+The `TemplateModel` as well as the `Request` and `Response` are types which are specified using the [Concerto modeling language](https://github.com/hyperledger/composer-concerto).
````
Missed one link update!

daniloff200 replied:
OOOOOOOOOOOOOOOOHHHHHHH

daniloff200 replied:
You hadn't seen this :rofl:
